docs/dev/pos/gui.rst

```diff
@@ -6,7 +6,7 @@ action_button
 
 Here you will find explanation of how to get/inherit action_button POS objects.
 
-For example we have definition in /../../../*js file::
+For example we have definition in /odoo9/addons/pos_reprint/static/src/js/reprint.js file::
```

Comment:
For example we have definition in `reprint.js <https://github.com/odoo/odoo/blob/9.0/addons/pos_reprint/static/src/js/reprint.js>`_ file::